compiler/cli/src/commonTest/kotlin/community/flock/wirespec/compiler/cli/MainTest.kt

```diff
@@ -0,0 +1,74 @@
+package community.flock.wirespec.compiler.cli
+
+import community.flock.wirespec.compiler.cli.io.Extension
+import community.flock.wirespec.compiler.cli.io.File
+import community.flock.wirespec.compiler.cli.io.FullFilePath
+import community.flock.wirespec.compiler.core.emit.common.DEFAULT_PACKAGE_NAME
+import kotlin.test.Test
+import kotlin.test.Ignore
+import kotlin.test.assertEquals
+
+
+class CliTest {
+
+    fun outputDir() = "/tmp/${getRandomString(8)}"
+    val inputDir = "src/commonTest/resources"
+
+    class TestFile(val directory: String, val fileName: String, val extension: Extension) : File(FullFilePath(directory, fileName, extension)) {
+        override fun copy(fileName: String) = TestFile(directory, fileName, extension)
+    }
+
+    @Test
+    @Ignore
+    fun testCliHelp() {
+        cli(arrayOf("--help"))
+    }
+
+    @Test
+    fun testCliOutput() {
+        val packageDir = DEFAULT_PACKAGE_NAME.replace(".", "/")
+        val outputDir = outputDir()
+        cli(arrayOf(inputDir, "-o", outputDir))
```

Comment:
This is a test with a side effect on disk. Is it guaranteed that '/tmp' is available and cleaned up on *nix systems? I don't think this is the case

Reply:
Thats why I create an random folder in tmp

Reply:
Goeie